_layouts/default.html

```diff
@@ -143,6 +145,13 @@
                 </a>
               </li>
             {% endfor %}
+            {% if site.enable_switch_theme_button == true %}
+            <li class="aux-nav-list-item">
+              <a href="#" class="site-button color-scheme-switch-theme-button">
+                <svg aria-hidden="true" class="site-button-icon"><use href="#svg-{{ site.color_scheme | default: 'light' }}"></use></svg>
```

Comment:
I think it is correct if there is no alternative text.

Reply:
Hm, so right now this button screenreader-accessible text at all. This means that users would have no idea what this button does.

In this case, we should have some sort of label - for example, an `aria-label` on the `svg` itself - that indicates the **purpose** of the button (opposed to describing the image). For example, instead of "a sun", it should be "switch to light mode". Does this make sense?

Reply:
If you see the `switch_color_scheme.html` each symbol has a title... i think is sufficent(?) 

P.S. I am just speculating...

Reply:
OK after reading [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)'s docs I agree with you. 

Sorry for the delay I was sure the title in the icon was sufficient.

Reply:
Hm, so now I see it's a generic `aria-label` regardless of the state of the button. I don't think this is accessible enough; I would prefer text that indicated both the current theme and the theme it'd switch to. You are either going to want to:

- add some Jekyll/JS code to the `button`'s `aria-label`;
- follow my original suggestion; remove the `aria-hidden=true` on the original `svg` element, and better annotate the icons

Reply:
go for this:
> follow my original suggestion; remove the `aria-hidden=true` on the original svg element, and better annotate the icons

Reply:
In that case, you just want to annotate each individual SVG (right now you're treating them all as the same blanket icon, when in reality they change depending on the page state). The `aria-label` docs you linked above mention various ways to annotate items like SVGs.

Reply:
ok